libs/platform/src/lib/components/upload-collection/upload-collection/upload-collection.component.scss

```diff
@@ -0,0 +1,203 @@
+@mixin fd-reset {
+    font-size: var(--sapFontSize);
+    line-height: var(--sapContent_LineHeight);
+    color: var(--sapTextColor);
+    font-family: var(--sapFontFamily);
```

Comment:
Great job using variables there, Could you add the declaration with value, before each? So
from:
`font-size: var(--sapFontSize);` 
To:
```
font-size: 0.75rem;
font-size: var(--sapFontSize);
```
Otherwise IE11 won't know what's going on

Reply:
Hi @JKMarkowski, thank you for the comments.

what about if it is a color variable?
from which theme should I use color?